defaults/main.yml

```diff
@@ -6,41 +6,41 @@ graylog_java_repo:                 "deb http://ppa.launchpad.net/webupd8team/jav
 graylog_java_src_repo:             "deb-src http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main"
 graylog_java_repo_keyserver:       "keyserver.ubuntu.com"
 graylog_java_repo_key:             "EEA14886"
-graylog_version:                   "2.4"
+graylog_version:                   "3.0"
 graylog_package_state:             "present"
-graylog_apt_deb_url:               "https://packages.graylog2.org/repo/packages/graylog-{{ graylog_version }}-repository_latest.deb"
+graylog_apt_deb_url:               "https://packages.graylog2.org/repo/packages/graylog-3.0-repository_latest.deb"
```

Comment:
Can't we continue to use the `graylog_version` variable here? I think that would be nice so a user only has to change the one setting to get the new version.

Reply:
Gotcha,but that might create some more work, as there are a `graylog_version` and a `graylog_server_version` var. I'll try to resolve this.

Reply:
Just my 2 cents as a bystander who had to set up graylog v3 system as well and stumbled upon this PR:

AFAICT `graylog_version` is handling the "major version"/base Debian repository (e.g. 2.4 vs 3.0), while `graylog_server_version` is handling the very specific Debian package to be used (3.0.1-2 vs 3.0.2-1). So keeping it as `graylog_version` as it used to be and defaulting this to 3.0 instead should continue to work and provide the identical behavior as it used to.

A user might indeed configure `graylog_version: 2.4` and use `graylog_server_version: 3.1.0` and this will fail (BTDT), but this behavior was also there before this PR. :)

HTH